src/accordion/section.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import uniqueId from '../utils/unique-id.js';
+import styles from './section.css';
+
+const headings = {
+  1: 'h1',
+  2: 'h2',
+  3: 'h3',
+  4: 'h4',
+  5: 'h5',
+  6: 'h6',
+};
+
+export default class Section extends React.Component {
+  constructor() {
+    super();
+    this.accessibleId = uniqueId();
+  }
+
+  render() {
+    const {
+      cannotClose,
+      children,
+      cssContainer,
+      cssHeading,
+      cssPanel,
+      cssTrigger,
+      headingLevel,
+      onClick,
+      open,
+      title,
+    } = this.props;
+    const titleVal = typeof title === 'function' ? title({ open }) : title;
+    const Heading = headings[headingLevel];
+
+    if (!Heading) {
+      throw new Error('Accordion must have a headingLevel between 1 and 6');
```

Comment:
Fun fact: if a user passes in `'toString'` it'll get past this check (but will still fail proptypes) and it'll even render something (`[object Object]` probably?)

And if they reaaaally wanted to, they could:
```jsx

Object.prototype.Cheat = function Cheat() {
  return <div> I hate headings and buttons </div>
};

<Section headingLevel='Cheat' ...>
```

Just thought that was an funny work around 😜 I don't think we need to account for that at all.

Reply:
Really interesting point, though. I forgot that objects have some other properties on them because of the prototype.

Seems like a `hasOwnProperty` check instead wouldn't be that bad, and would prevent this edge case.